### ISSUE TITLE
Unable to get profile info from other users

### ISSUE THREAD

**Issue by justbill2020** [cult-of-coders/grapher#281]
I've created a reducer 

``` javascript
Meteor.users.addReducers({
  email: {
      body: {
          emails: {
              address: 1,
              verified: 1
          }
      },
      reduce(object) {
          const {emails} = object;
          if (!emails) return 'No Email Found'
          let result = emails.find(e=>{
            return e.verified
          }).address
          if (result) return result
          return `PLEASE VERIFY EMAIL`;
      }
  }
})
```

The issue is that for users other than the current user i'm logged in as always provides the `_id` within object but no other information. I'm sure there's something i'm missing however i don't know where to start. It seems I'm not exposing Users but not sure the proper method to expose that data without creating a named query or accidentally over exposing data.

**Comment by theodorDiaconu** [comment]
@justbill2020 it's very likely it's an exposure issue, I need to know all the details, how you query, how you expose, maybe a quick reproduction ?

**Comment by justbill2020** [comment]
That’s the issue I’m not sure how to properly expose the users... I have a collection called games, it has a field players which is an array of user ids. I then have created a link to the users collection. And reducers for the email address and full name. It all works for the logged in user however not for other users. How do I properly expose the users collection so that this works? I’m trying to stay away from pub/sub functionality of meteor.

**Comment by theodorDiaconu** [comment]
Can you please share with me the code ? It's hard to help you debug this.
Maybe a reproduction repo ?

**Comment by theodorDiaconu** [comment]
@justbill2020  any updates ?

**Comment by justbill2020** [comment]
@theodorDiaconu can we set up a time to go over it? I don't know how to explain in details here... i can add you to the private repo to try and help... I seem to be exposing it but for whatever reason it doesn't seem to expose anything other than the id of the Meteor.users collection. Have you done this before and how did you do it?

**Comment by justbill2020** [comment]
ok i finally found the issue with exposure but i think there's a similar issue... I am using withQuery. I'm getting multiple documents from the query and the first item uses reducers correctly to display the following user information. 
```JSON
{
  "_id": "test",
  "players": [
    "NeRFemkdw2s79yMgN",
    "QK6QXWs6HHpPyRcr8"
  ],
  "playerNames": [
    {
      "_id": "QK6QXWs6HHpPyRcr8",
      "emails": [
        {
          "address": "bill@something.com",
          "verified": true
        }
      ],
      "fullName": "first1 last1",
      "email": "bill@something.com"
    },
    {
      "_id": "NeRFemkdw2s79yMgN",
      "emails": [
        {
          "address": "mike@something.com",
          "verified": true
        }
      ],
      "fullName": "First2 Last2",
      "email": "mike@something.com"
    }
  ],
}
```
the second is has this in the same linked data field:
```JSON
{
  "_id": "game",
  "players": [
    "NeRFemkdw2s79yMgN",
    "QK6QXWs6HHpPyRcr8"
  ],
  "playerNames": [
    {
      "_id": "QK6QXWs6HHpPyRcr8",
      "emails": [
        {
          "address": "bill@something.com",
          "verified": true
        }
      ],
      "profile": {
        "firstName": "first1",
        "lastName": "Last1"
      }
    },
    {
      "_id": "NeRFemkdw2s79yMgN",
      "emails": [
        {
          "address": "mike@something.com",
          "verified": true
        }
      ],
      "profile": {
        "firstName": "First2",
        "lastName": "Last2"
      }
    }
  ]
}
```

**Comment by theodorDiaconu** [comment]
@justbill2020 can you try using a scoped query ? They have been released today in 1.3.9.
